designSystem/src/main/java/com/example/designsystem/components/custom_snack_bar/SnackBar.kt

```diff
@@ -0,0 +1,94 @@
+package com.example.designsystem.components.custom_snack_bar
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.clipToBounds
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.example.designsystem.R
+import com.example.designsystem.theme.AflamiTheme
+import com.example.designsystem.theme.AppTheme
+import com.example.designsystem.utils.ThemeAndLocalePreviews
+import com.example.designsystem.utils.dropShadow
+
+@Composable
+fun BoxScope.SnackBar(message: String, status: SnackBarStatus) {
```

Comment:
Compose function should have modifier to make it more reusable and make it without BoxScope may be we need to use it in another scope:
```suggestion
fun SnackBar(
 message: String,
 status: SnackBarStatus,
 modifier: Modifier = Modifier
) {
```

Reply:
Snack bar doesn't need to be modified across the app, so no need for modifier